habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -1296,6 +1296,7 @@ class SimulatorConfig(HabitatBaseConfig):
     auto_sleep: bool = False
     step_physics: bool = True
     concur_render: bool = False
+    enable_batch_renderer: bool = False
```

Comment:
How about document this? Or point to somewhere where you're explaining how to use the batch renderer?